BUILD.md

```diff
@@ -18,6 +18,7 @@ To build the project and run the tests, simply run:
 mvn package
 ```
 
+NB: if the build fails, you may need to clean previously built artifacts with the `mvn clean` commands.
```

Comment:
I needed to whitelist you for the build to pass, to trigger a build you'll need to push another commit so here is a tiny suggestion.

```suggestion
NB: if the build fails, you may need to clean previously built artifacts with the `mvn clean` command.
```